source/contracts/extensions/MarketFeeCalculator.sol

```diff
@@ -15,42 +15,65 @@ contract MarketFeeCalculator {
     mapping (address => uint256) private validityBondInAttoeth;
     mapping (address => uint256) private targetReporterGasCosts;
     mapping (address => uint256) private designatedReportStakeInAttoRep;
+    mapping (address => uint256) private designatedReportNoShowBondInAttoRep;
 
     uint256 private constant TARGET_INVALID_MARKETS_DIVISOR = 100; // 1% of markets are expected to be invalid
     uint256 private constant TARGET_REP_MARKET_CAP_MULTIPLIER = 5;
 
     uint256 private constant TARGET_INCORRECT_DESIGNATED_REPORT_MARKETS_DIVISOR = 100; // 1% of markets are expected to have an incorrect designate report
 
-    function getValidityBond(IReportingWindow _reportingWindow) public returns (uint256) {
+    uint256 private constant TARGET_DESIGNATED_REPORT_NO_SHOWS_DIVISOR = 100; // 1% of markets are expected to have an incorrect designate report
+
+    function getValidityBond(IUniverse _universe) public returns (uint256) {
+        IReportingWindow _reportingWindow = _universe.getCurrentReportingWindow();
```

Comment:
I'm curious why you decided to change this to take a `universe` instead of `reportingWindow`?  It appears you only use universe to get the reporting window, suggesting that you could just take in the reporting window instead?

Reply:
Same a couple other times below.

Reply:
Since we always want these to use the universe's current reporting window if we took in a reporting window we'd have to get the universe from it and then get the current reporting window

Reply:
Ah, I see.  This allows us to not have to trust the caller to provide current reporting window.  👍